core/templates/pages/preferences-page/preferences-page.component.css

```diff
@@ -0,0 +1,72 @@
+/*
+  NOTE to developers: We generate RTL CSS from this file in the webpack
+  compilation, here are some additional rules that can be added to the CSS files:
+  https://rtlcss.com/learn/usage-guide/control-directives .
+*/
+
+.oppia-dashboard-container .checkbox label {
+  font-weight: normal;
+}
+
+.oppia-page-card {
+  text-align: left;
+}
+
+.oppia-preferences-page-heading {
+  color: #01645c;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+  font-size: 3em;
+  margin-bottom: 10px;
+  margin-top: 0;
+  padding-top: 45px;
+  position: relative;
+  text-align: center;
+}
+
+.oppia-preferences-page-heading-subtext {
+  color: #01645c;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+  font-size: 1.2em;
+  position: relative;
+  text-align: center;
+}
+
+.oppia-bio-border {
+  border-color: rgb(156,156,156);
+  border-radius: 4px;
+  border-width: 0.01em;
+}
+
+.oppia-profile-picture-container {
+  height: 150px;
+  width: 150px;
```

Comment:
![Screenshot from 2022-09-06 14-13-19](https://user-images.githubusercontent.com/24855641/188589993-2f964adf-81c6-42e8-a153-d2633140abdc.png)
@aks681 This has caused a UI bug. PTAL and also https://github.com/oppia/oppia/pull/15990

Reply:
This problem has already been fixed here PR: https://github.com/oppia/oppia/pull/15990#issuecomment-1237534884
Issue: https://github.com/oppia/oppia/issues/15988